### PR TITLE
qemu_storage: add repair option for image check function

### PR DESCRIPTION
**Pull request by hellohellenmao**
Signed-off-by: Tingting Mao <timao@redhat.com>

ID: 2103037

### CODE REVIEW

**Comment by hellohellenmao** [comment]
(1/2) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.qemu_img_check_data_integrity.kill_qemu_during_io.q35: STARTED
 (1/2) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.qemu_img_check_data_integrity.kill_qemu_during_io.q35: PASS (360.65 s)
 (2/2) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.qemu_img_check_data_integrity.kill_qemu_after_io.q35: STARTED
 (2/2) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.qemu_img_check_data_integrity.kill_qemu_after_io.q35: PASS (365.88 s)

**Comment by hellohellenmao** [comment]
@zhencliu  Could you please help to take a review? Thanks.

**Comment by zhencliu** [comment]
LGTM
hi @YongxueHong , would you please make a double check? Thanks.

File: virttest/qemu_storage.py [hunk 1]
```diff
@@ -1534,10 +1534,12 @@ def check(self, params, root_dir, force_share=False, output=None):
         image_filename = self.image_filename
         LOG.debug("Checking image file %s", image_filename)
         force_share &= self.cap_force_share
+        check_repair = self.params.get("image_check_repair")
```
Comment (YongxueHong) [inline virttest/qemu_storage.py:1537]: Hi @zhencliu @hellohellenmao 
I suggest adding a new argument "check_repair=None" for the definition.
`def check(self, params, root_dir, force_share=False, output=None, check_repair=None):`
From my understanding, the attribute `check_repair` doesn't belong to the image namespace, not like `image_size`, `image_format`, `image_filename`, and so on.
The `check_repair` just focuses on the operation checking of the image, it's not a common attribute. So I think shall we add a new argument for this options?
Thanks.

Comment (hellohellenmao) [reply virttest/qemu_storage.py:1537]: @zhencliu What do you think?

Comment (zhencliu) [reply virttest/qemu_storage.py:1537]: hi @YongxueHong , agreed. I considered to add a new arg, I thought that once we want to repair an image in test cases, we have to update py files to give a new arg. But I never considered that if check_repair is a proper attribute of an image, yes, it's not an attribute of an image.  hi @hellohellenmao , what's your opinion?

Comment (hellohellenmao) [reply virttest/qemu_storage.py:1537]: Agree, and okay from my side. I will update it soon. Thanks all.

**Comment by hellohellenmao** [comment]
(1/2) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.qemu_img_check_data_integrity.kill_qemu_during_io.q35: STARTED
 (1/2) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.qemu_img_check_data_integrity.kill_qemu_during_io.q35: PASS (370.76 s)
 (2/2) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.qemu_img_check_data_integrity.kill_qemu_after_io.q35: STARTED
 (2/2) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.qemu_img_check_data_integrity.kill_qemu_after_io.q35: PASS (372.50 s)

**Comment by hellohellenmao** [comment]
@zhencliu @YongxueHong Could you please help to review again? Thanks.

**Review by YongxueHong — APPROVED** [review]

**Pull request merged**